modules/Vtiger/uitypes/Base.php

```diff
@@ -576,4 +576,26 @@ public function getFieldInfo(): array
 	{
 		return $this->getFieldModel()->loadFieldInfo();
 	}
+
+	public function isColumnLengthChangeAllowed(): bool
+	{
+		return true;
+	}
+
+	public function validateColumnLength($newColumnLength): bool
+	{
+		$minColumnLength = 0;
+		$maxColumnLength = 255;
+		$newColumnLength = (int) $newColumnLength;
+		if ($newColumnLength > $minColumnLength && $newColumnLength <= $maxColumnLength) {
+			return true;
+		}
```

Comment:
?

Reply:
Make sure that string is Base, create uitype fo string? Check system field for don't edit length. Return on end of method. True or false on return without throw